nextflow.config

```diff
@@ -0,0 +1,193 @@
+/*
+ * -------------------------------------------------
+ *  nf-core/deepvariant Nextflow config file
+ * -------------------------------------------------
+ * Default config options for all environments.
+ * Cluster-specific config options should be saved
+ * in the conf folder and imported under a profile
+ * name here.
+ */
+
+ // Global default params, used in configs
+ params {
+
+   container = 'nfcore/deepvariant:dev'
+
+   help = false
+   outdir = 'results'
+   email = false
+
+   //BAM files
+   bam=""
+   bam_folder=""
+   bam_file_prefix=""
+   getBai=false
+
+   // Reference genomes
+   genomes_base = 's3://deepvariant-data/genomes'
+   //genome = 'hg19' //if no fasta specified default of hg19 will be used (see main.nf)
+
+   // Fasta files
+   fasta=false
+   fai=false
+   fastagz=false
+   gzfai=false
+   gzi=false
+
+   // Exome data
+   exome=""
+   bed=""
+
+   // Model
+   modelBase="s3://deepvariant-data"
+   modelFolder="${modelBase}/models"
+   modelName="model.ckpt"
+
+   //Params for the Read Group Line to be added just in case its needed.
+   rgid=4
+   rglb="lib1"
+   rgpl="illumina"
+   rgpu="unit1"
+   rgsm=20
+
+   tracedir = "${params.outdir}/pipeline_info"
+   clusterOptions = false
+   awsqueue = false
+   awsregion = 'eu-west-1'
+   // TODO: Remove this if/when we can. See https://github.com/nextflow-io/nextflow/issues/840
+   pipelineVersion = '1.0dev'
+ }
+
+process.container = 'nfcore/deepvariant:dev'
+
+docker {
+  enabled=true
+}
+
+profiles{
+
+  s3{
```

Comment:
I think these first few configs are all for test data.. Maybe put all of these at the end of the `profile` section and begin them all with a `test_` prefix. eg. `test_local`, `test_s3wholeGenome`, `test_s3exome` so that it's clear what's happening with them.

Also, I think you can probably reduce them to two lines by using the new file path base variables?